README.md

```diff
@@ -231,7 +232,8 @@ See the file config.example.yaml for an example config file
 | [image_date_format](#date-format) | KIOSK_IMAGE_DATE_FORMAT | string                     | DD/MM/YYYY  | The format of the image date. default is day/month/year. See [date format](#date-format) for more information. |
 | show_image_exif                   | KIOSK_SHOW_IMAGE_EXIF   | bool                       | false       | Display image Fnumber, Shutter speed, focal length, ISO from METADATA (if available).      |
 | show_image_location               | KIOSK_SHOW_IMAGE_LOCATION | bool                     | false       | Display the image location from METADATA (if available).                                   |
-| show_image_id                     | KIOSK_SHOW_IMAGE_ID     | bool                       | false       | Display the image Immich ID.                                   |
+| hide_countries                    | KIOSK_HIDE_COUNTRIES    | []string                   | []          | List of countries whose name will be hidden                                                |
```

Comment:
Make this a little easier to read and give context. "List of countries to hide from image_location"